com.unity.render-pipelines.high-definition/Documentation~/Accumulation.md

```diff
@@ -115,6 +115,115 @@ You can easily define the first three profiles without using an animation curve
 
 In this example, you can see that the slow open profile creates a motion trail appearance for the motion blur, which might be more desired for artists. On the other hand, the smooth open and close profile creates smoother animations than the slow open or uniform profiles.
 
+
+## High Quality Anti-aliasing wiht Accumulation
+You can also use the accumulation API to get a high quality antialiased frame by jittering the projection matrix of each renderer subframe. This is equivalent to rendering a higher resolution image and then downscaling to the final resolution, a process that is often callen **SuperSampling**. Using the accumulation API is preferable over rendering at higher resolutions, since it is using a much lower memory footprint. The following example shows how to properly setup the accumulation API from a script to perform high quality antialiasing:
```

Comment:
```suggestion
You can also use the accumulation API to get a high quality antialiased frame by jittering the projection matrix of each renderer subframe. This is equivalent to rendering a higher resolution image, then downscaling to the final resolution, a process that is often called **SuperSampling**. Using the accumulation API is preferable to rendering at higher resolutions, since it is using a much lower memory footprint. The following example shows how to properly set up the accumulation API from a script to perform high quality antialiasing:
```

Reply:
What is doing the jittering of the projection matrix?

Reply:
The jittering is done by the user script. In the example that I have included it's done by the GetJitteredProjectionMatrix() method.